verify/verify_test.go

```diff
@@ -109,12 +110,12 @@ func TestVerifyVcekCert(t *testing.T) {
 	if err != nil {
 		t.Errorf("could not parse valid VCEK certificate: %v", err)
 	}
-	opts := root.X509Options()
+	now := time.Date(2022, time.September, 24, 1, 0, 0, 0, time.UTC)
```

Comment:
Just curious: any reasons why this date is used?

Reply:
It was about the timestamp of my machine when I first wrote the test case.